tests-and-examples/tests/tests.rs

```diff
@@ -1,3 +1,7 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+#[cfg(feature = "protobuf-codec")]
 mod cases;
+
+#[cfg(feature = "protobufv3-codec")]
+mod cases_protobufv3;
```

Comment:
Tests should be updated in place. Otherwise it's hard to maintain two copies for every test cases.